pepper-apis/src/main/java/org/broadinstitute/ddp/DataDonationPlatform.java

```diff
@@ -251,6 +262,15 @@ private static void start() {
 
         SimpleJsonTransformer responseSerializer = new SimpleJsonTransformer();
 
+        int maxQueriesPerSecond = DEFAULT_RATE_LIMIT_MAX_QUERIES_PER_SECOND;
+        int burst = DEFAULT_RATE_LIMIT_BURST;
+        if (cfg.hasPath(ConfigFile.API_RATE_LIMIT.MAX_QUERIES_PER_SECOND)) {
+            maxQueriesPerSecond = cfg.getInt(ConfigFile.API_RATE_LIMIT.MAX_QUERIES_PER_SECOND);
```

Comment:
Just took a peek at vault. We actually have a string for this value: `10r/s`. `cfg.getInt()` will fail parsing that. Quick remedy is to update vault once we roll this out.

Reply:
Oh, just saw updated PR description. Resolving.